docroot/themes/custom/mass_theme/overrides/css/basic-html.css

```diff
@@ -115,3 +115,12 @@ form.fsForm.rtl {
 .page-content--intro .ma__video__skip-link {
   background-color: #fff;
 }
+
+.ma__rich-text h4 {
```

Comment:
Can you move this into Mayflower? Also we should clear float for all heading levels, e.g. H2, H3, etc.
This is an example taken from Mayflower:
![Screenshot 2024-08-22 at 5 32 24 PM](https://github.com/user-attachments/assets/2adcb2d9-48e0-4682-a22d-cdc49d4834b4)


In mayflower packages/assets/scss/03-organisms/_rich-text.scss
```
// clear float for all headings after left of right floating figures 
.ma__rich-text {
  h1, h2, h3, h4, h5, h6 {
     clear: both;
}
}
```

Reply:
Added to the PR https://github.com/massgov/mayflower/pull/1930 .